Update Debug Tracing doc (#22)
AdobeDocs/adobe-io-events: src/pages/support/tracing.md

> # Tracing Adobe I/O Events

Reviewer: Can you also update the snapshot at https://developer.adobe.com/events/docs/guides/#asynchronous-validation?

Author: Good catch. Updated.

For help in debugging your webhook-based event-registrations, Adobe offers Debug Tracing. Using Debug Tracing, you can inspect the activity between the Adobe service and your integration to see what data is being sent and how your integration is responding.

Each trace comprises of the event delivery HTTP request (sent by I/O Events), and response by the consumer application. This information can be of critical importance to your debugging efforts.

**To access Debug Tracing:**

1. Log into [Adobe Developer Console](https://developer.adobe.com/console/) and select the project containing the event registration.

2. From the *Project Overview*, select the event under *Events* in the left navigation.

3. From the *Registration Details*, select the *Debug Tracing* tab. You will see a list of events, beginning with the most recent.

> 4. Select *Debug Tracing* tab. This section shows last 100 traces for your webhook registration. 

Reviewer: We already have selected the "Debug Tracing" tab in step 3.
```suggestion
4. The *Debug Tracing* tab shows last 100 traces for your webhook registration. 
```
